### PR TITLE
issue-2-view-agent-details

### PR DESCRIPTION
**Pull request by jeff-cruz**
Kapture 2022-04-20 at 11 24 40

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -2,30 +2,203 @@ var $start = document.querySelector('.start-button');
...
+          $abilityThreeText.textContent = xhr.response.data[i].abilities[2].description;
+
+          $abilityFourIcon = document.querySelector('.ability-four-icon');
+          $abilityFourIcon.setAttribute('src', xhr.response.data[i].abilities[3].displayIcon);
+
+          $abilityFourName = document.querySelector('.ability-four-name');
+          $abilityFourName.textContent = xhr.response.data[i].abilities[3].displayName;
+
+          $abilityFourText = document.querySelector('.ability-description-four');
+          $abilityFourText.textContent = xhr.response.data[i].abilities[3].description;
```
Comment (codingcodymiller) [inline js/main.js]: Why not do this with a loop over the `abilities` array?

File: js/main.js [hunk 2]
```diff
@@ -2,30 +2,203 @@ var $start = document.querySelector('.start-button');
...
+
+          var $abilityFourName = document.createElement('p');
+          $abilityFourName.setAttribute('class', 'ability-name ability-four-name noto-font');
+          $abilityFourName.textContent = xhr.response.data[i].abilities[3].displayName;
+          $abilityFourContainer.appendChild($abilityFourName);
+
+          var $abilityFourText = document.createElement('p');
+          $abilityFourText.setAttribute('class', 'ability-description ability-description-four');
+          $abilityFourText.textContent = xhr.response.data[i].abilities[3].description;
+          $abilityFourContainer.appendChild($abilityFourText);
```
Comment (codingcodymiller) [inline js/main.js]: Why not do this with a loop?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js [hunk 3]
```diff
@@ -62,140 +62,143 @@ function agentDetails(event) {
...
+              $abilityTwoText.textContent = xhr.response.data[id].abilities[j].description;
+            } else if (j === 2) {
+              $abilityThreeIcon.setAttribute('src', xhr.response.data[id].abilities[j].displayIcon);
+              $abilityThreeName.textContent = xhr.response.data[id].abilities[j].displayName;
+              $abilityThreeText.textContent = xhr.response.data[id].abilities[j].description;
+            } else if (j === 3) {
+              $abilityFourIcon.setAttribute('src', xhr.response.data[id].abilities[j].displayIcon);
+              $abilityFourName.textContent = xhr.response.data[id].abilities[j].displayName;
+              $abilityFourText.textContent = xhr.response.data[id].abilities[j].description;
+            }
```
Comment (codingcodymiller) [inline js/main.js]: This is still the same problem. You should be able to make the elements, configure the elements, and append the elements inside of this loop. There should be no conditional necessary.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by jeff-cruz** [comment]
Kapture 2022-04-20 at 15 18 24

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**